tempel.el

```diff
@@ -144,6 +144,21 @@ may be named with `tempel--name' or carry an evaluatable Lisp expression
     ((or 'n 'n> '> '& '% 'o) " ")
     (_ "_")))
 
+(defun tempel--expand-locations (locations)
+  "Return the list of files specified by LOCATIONS.
+LOCATIONS is a list of files and directories."
+  (setq locations (if (listp locations) locations (list locations)))
+  (let (files)
+    (dolist (file-or-dir locations)
+      (if (file-directory-p file-or-dir)
+	  (cl-callf append files (directory-files file-or-dir t))
+	(push file-or-dir files)))
+    (mapcar #'expand-file-name (reverse files))))
+
+(defun tempel--load-templates ()
```

Comment:
Maybe call this `tempel--load-path-templates`? And merge the function together with `tempel--expand-locations`, I don't see much value in having these functions separate.